model_compression_toolkit/core/keras/back2framework/keras_model_builder.py

```diff
@@ -303,10 +302,9 @@ def _run_operation(self,
                 # for the activation quantization holder
                 if self.use_activation_holder_during_model_building:
                     activation_quantizer_holder = self.get_activation_quantizer_holder(n)
+                    out_tensors_of_n = activation_quantizer_holder(out_tensors_of_n)
```

Comment:
I think something is missing, what if we don't have a wrapper but we want to use the holder for activations? why do they need to be coupled like this? (I had something else in mind when we talked about this code yesterday).

Also, if the code for input is similar to the code for the other types of nodes, please extract it to a function, so we won't need to modified twice each time we change something here.